README.md

```diff
@@ -69,15 +78,15 @@ if($player instanceof Player && $player->isOnline()){
 	 * @phpstan-param Vector3|null $facing_pos
 	 */
 	CameraInstruction::set(
-		preset: libcamera::getPresetRegistry()->registered["target"],
+		preset: CameraPresetRegistry::TARGET(),
 		ease: new CameraSetInstructionEase(
 			CameraSetInstructionEaseType::IN_OUT_CUBIC,
 			(float) 5.0 // duration (sec)
 		),
-		camera_pos: null,
+		camera_pos: $player->getPosition()->add(0.0, $player->getEyeHeight(), 0.0), //Without it, the camera will teleport into subspace
 		rot: new CameraSetInstructionRotation(
-			(float)20.0, //pitch
-			(float)180.0 //yaw
+			(float)20, //pitch
```

Comment:
unnecessary change which will make phpstorm and probably phpstan complain (also inconsistent with other occurrences of floating point numbers, see line 84 for example)